run

```diff
@@ -52,41 +58,41 @@ ________________________________________________________________________________
 
 Development Quick Start Guide:
 
-$(bold ./$name pnpm install)           # Install packages
-$(bold ./$name dev)                    # Launch dev server with examples, open http://localhost:9000
-$(bold ./$name docs:dev)               # Launch official website, open http://localhost:3333
+$(bold ./"$name" pnpm install)           # Install packages
+$(bold ./"$name" dev)                    # Launch dev server with examples, open http://localhost:9000
+$(bold ./"$name" docs:dev)               # Launch official website, open http://localhost:3333
 
-$(bold ./$name pnpm vitest)            # Run watcher for unit tests
-$(bold ./$name cypress)                # Run integration tests (after starting dev server)
-$(bold ./$name pnpm build)             # Prepare it for production
+$(bold ./"$name" pnpm vitest)            # Run watcher for unit tests
+$(bold ./"$name" cypress)                # Run integration tests (after starting dev server)
+$(bold ./"$name" pnpm build)             # Prepare it for production
```

Comment:
although looks odd, it is the only way to make it robust if script would ever be renamed to include space in the name. Alternatively that check could be skipped indeed.